lib/multitenancy/headers.js

```diff
@@ -39,6 +39,7 @@ export default function (pluginRoot, server, kbnServer, APP_ROOT, API_ROOT, auth
     const private_enabled = config.get('opendistro_security.multitenancy.tenants.enable_private');
     const preferredTenants = config.get('opendistro_security.multitenancy.tenants.preferred');
     const debugEnabled = config.get('opendistro_security.multitenancy.debug');
+    const unauthenticatedRoutes = config.get('opendistro_security.auth.unauthenticated_routes');
```

Comment:
what if there is no such config item, and unauthenticatedRoutes is null?

Reply:
Please check https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/master/index.js#L44 
If undefined, it is added as default in config. Let me know if null check is needed